gradlew

```diff
@@ -56,9 +56,9 @@ while [ -h "$PRG" ] ; do
     fi
 done
 SAVED="`pwd`"
-cd "`dirname \"$PRG\"`/" >&-
+cd "`dirname \"$PRG\"`/" >/dev/null
```

Comment:
Just to point this out. While changing this line, I would have liked to see the backticks removed by dollar sign parens notation. it's pretty much *always* better to use "$(..)" syntax instead of backtick syntax for in-line evaluation. "$(..)" is easier to see, and always nests properly.

IE:
+cd "$(dirname \"$PRG\")/" >/dev/null
instead of:
+cd "`dirname \"$PRG\"`/" >/dev/null